tests/CatalogTests/Helpers/FeedHelpersTests.cs

```diff
@@ -267,10 +405,58 @@ private static IEnumerable<ODataPackage> ODataPackages
                         Published = new DateTime(2017, 5, 1, 0, 0, 0),
 
                         LicenseNames = "ABCD",
-                        LicenseReportUrl = "http://www.license.com"
+                        LicenseReportUrl = "https://unit.test/license"
                     }
                 };
             }
         }
+
+        private static string CreateCatalogJson(string propertyName, DateTimeOffset propertyValue)
+        {
+            return $"{{\"{propertyName}\":\"{(propertyValue.ToString("O"))}\"}}";
+        }
+
+        private static DateTimeOffset CreateDateTimeOffset(TimeSpan offset)
+        {
+            var now = DateTime.Now;
+            var datetime = new DateTime(
+                now.Year,
+                now.Month,
+                now.Day,
+                now.Hour,
+                now.Minute,
+                now.Second,
+                now.Millisecond,
+                DateTimeKind.Unspecified);
```

Comment:
Could be written shorter: `var datetime = new DateTime(DateTime.Now.Ticks, DateTimeKind.Unspecified);`